transcom/mymove
# INT-B-18637 create bb order type

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A904624&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Added BLUEBARK order type as well as labels on all Office Queues and on the Move Details header for office users and customers.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the customer app
2. Login as a new user
3. Create a move with the BLUEBARK order type
4. Test BLUEBARK label on customer home page header
5. Login as a TOO/TIO/SC/QAE and search for new move
6. Make sure BLUEBARK label is visible within the move queue and on the Move Details Header
